examples/mugen/retrieval/video_clip.py

```diff
@@ -6,41 +6,54 @@
 
 import torch
 from torch import nn
-from torchmultimodal.utils.common import get_current_device
-from transformers import DistilBertConfig, DistilBertModel, DistilBertTokenizer
+from transformers import DistilBertConfig, DistilBertModel
 
 
 class TextEncoder(nn.Module):
-    """Encode text to a fixed-length vector
+    """Encode tokenized text to a fixed-length vector
```

Comment:
What are you referring to by "fixed length vector"? 

- For seq2seq model, "fixed length vector" refers to a fixed intermediate vector ("context vector") to encoder a sentence. 
- Transformer-based models like BERT learns the pair-wise correlation between the source and the target tokens and then uses the learnt correlation for prediction given a new sequence of inputs. I don't think there's a corresponding "fixed vector" in this case as the correlation is calculated amongst all pairs of source/target tokens.

Reply:
edited the docstring summary to explain the CLS token and that the encoding is the last hidden state of the CLS token